app/views/install_beta.scala.txt

```diff
@@ -238,7 +238,8 @@ echo "sdkman_curl_max_time=10" >> "$sdkman_config_file"
 echo "sdkman_debug_mode=false" >> "$sdkman_config_file"
 echo "sdkman_insecure_ssl=false" >> "$sdkman_config_file"
 echo "sdkman_rosetta2_compatible=false" >> "$sdkman_config_file"
-echo "sdkman_selfupdate_enable=true" >> "$sdkman_config_file"
+echo "sdkman_auto_update=true" >> "$sdkman_config_file"
```

Comment:
I know I'm a pain :grimacing:, but do you mind putting this one in alphabetical order like the rest?

Reply:
Sure!

Reply:
Excuse my OCD 😂